src/Snowflake/Connection.php

```diff
@@ -232,4 +238,36 @@ public function disconnect(): void
             odbc_close($this->connection);
         }
     }
+
+    /**
+     * @param resource $connectionId
+     * @return false|resource
+     */
+    private function doOdbcPrepare($connectionId, string $queryString)
+    {
+        return $this->getRetryProxy()->call(
+            function () use ($queryString, $connectionId) {
+                return odbc_prepare($connectionId, $queryString);
+            }
+        );
+    }
+
+    private function getRetryProxy(): RetryProxy
+    {
+        if (!$this->retryProxy) {
+            $proxy = new RetryProxy(
```

Comment:
Bylo by dobrý tomu nějaký způsobem předat logger a logovat to jako WARN abysme věděli jestli se to děje.